### ISSUE TITLE
Not getting verification mails

### ISSUE THREAD

**Issue by unpack5** [ethereum-magicians/scrolls#35]
I've tried three different providers but the mails don't arrive, therefore I can't register to the site.

Could you look into the queue and check if mails are being sent out? I made all three request in about the past hour.

**Comment by bmann** [comment]
@jpitts can you please take a look at this.

**Comment by pet3r-pan** [comment]
I just used gmail to sign up and received the confirmation email.

**Comment by bmann** [comment]
I asked the user to post here. Their particular account / emails weren’t getting through, across three email addresses.

**Comment by unpack5** [comment]
@bmann Could you please ping Jamie on another channel?

**Comment by jpitts** [comment]
@unpack5, I cannot find you in the https://ethereum-magicians.org user db, are you using a different name or GitHub user ID to log in?

**Comment by unpack5** [comment]
@jpitts yes, my main intended registration is under a different username (which I don't want to link to my Github).

My registration is still there since I can't register with the same name. After my login attempts, the site says I can't proceed because I need to verify ma mail. I can request a re-send, but nothing arrives. I'm using a reputable paid e-mail service, thus my frustration.

Is there a channel where I can contact you privately with my mail/username?

FYI, I've just logged in and requested a new verification mail moments before sending this comment.

**Comment by jpitts** [comment]
@unpack5 you can find me in Riot/Matrix at https://matrix.to/#/@jpitts:matrix.org.

**Comment by unpack5** [comment]
@jpitts I wrote you on Matrix on the 1st of February. Did you receive it?
